app/controllers/reports_controller.rb

```diff
@@ -69,4 +71,9 @@ def date_match(employee)
       end
     end
   end
+
+  def get_reports
+    return Report.all
```

Comment:
Remember that Ruby by default returns the last value that is placed in the def. So the return is not necessary.